### PR TITLE
[sailfish-browser] Position tabview when opened in landscape. Fixes JB#31994

### PR DESCRIPTION
**Pull request by pvuorela**

### CODE REVIEW

File: src/pages/components/Overlay.qml [hunk 1]
```diff
@@ -417,6 +417,11 @@ Background {
 
             onStatusChanged: browserPage.tabPageActive = (status == PageStatus.Active)
 
+            Component.onCompleted: {
```
Comment (rainemak) [inline src/pages/components/Overlay.qml]: Place of ```Component.onCompleted``` (attached properties) has not been clearly defined, my personal preference for ```Component.onCompleted``` would be at the end of the ```tabPage```.

The ```positionViewAtIndex``` looks like obsolete now as this completes after tabView. Please remove if not really needed.

Comment (pvuorela) [inline src/pages/components/Overlay.qml]: I don't understand the 'looks like obsolete' part. If I remove it here I have no changes. If I remove it from TabView I'm relying on undefined execution order being defined as I see it happening.

**Comment by pvuorela** [comment]
Moved signal handler position.

**Comment by rainemak** [comment]
LGTM

**Pull request merged**